src/base/rss/rss_session.cpp

```diff
@@ -529,14 +529,14 @@ void Session::setRefreshInterval(const int refreshInterval)
 
 std::chrono::seconds Session::fetchDelay() const
 {
-    return m_storeFetchDelay;
+    return std::chrono::seconds(m_storeFetchDelay);
```

Comment:
Wouldn't it work without repeating result type?
```suggestion
    return {m_storeFetchDelay};
```

Reply:
Should work, but less explicit. I don't object as this is a short function.